Packs/Core/Playbooks/playbook-Unsigned_and_unpopular_process_performed_an_injection.yml

```diff
@@ -0,0 +1,823 @@
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 560,
+          "y": 2610
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 3214ade0-7bba-484f-8945-3bc4367178a9
+    type: title
+    task:
+      id: 3214ade0-7bba-484f-8945-3bc4367178a9
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 770,
+          "y": 2780
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: 358ad811-3ae6-4e1d-826e-ba15c09f050c
+    type: title
+    task:
+      id: 358ad811-3ae6-4e1d-826e-ba15c09f050c
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "18"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 90,
+          "y": 1490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: a4e84519-ae9c-4cde-86db-4210bd57a617
+    type: condition
+    task:
+      id: a4e84519-ae9c-4cde-86db-4210bd57a617
+      version: -1
+      name: Approve the endpoint isolation
+      description: Requires the analyst approval for endpoint isolation.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "8"
+      Isolate:
+      - "13"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -300,
+          "y": 2240
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 324312f8-a792-4ff6-8046-848f554bdf15
+    type: regular
+    task:
+      id: 324312f8-a792-4ff6-8046-848f554bdf15
+      version: -1
+      name: Isolate endpoint
+      description: Isolates the specified endpoint.
+      script: '|||core-isolate-endpoint'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      endpoint_id:
+        simple: ${alert.agentid}
+      incident_id:
+        simple: ${parentIncidentFields.incident_id}
```

Comment:
Also here, please use 'alert.parentXDRIncident' instead of 'parentIncidentFields.incident_id'. 

You can use it as follows:

Get
alert.parentXDRIncident
Where
No filters applied
Transformers
Cut (delimiter: -, fields: 2)
Concat (prefix: caseid:, suffix: )